src/main/cpp/VisionIO.cpp

```diff
@@ -0,0 +1,32 @@
+#include "io/VisionIO.h"
+void VisionIO::Periodic() {
+ProcessIO();
```

Comment:
This should be called using `VOKC_CALL`

Reply:
the include file needs to be changed right

Reply:
Yeah you would need to include Utils.h